Content.Server/Fax/FaxMachineComponent.cs

```diff
@@ -13,60 +14,72 @@ public sealed partial class FaxMachineComponent : Component
     /// Name with which the fax will be visible to others on the network
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite)]
-    [DataField("name")]
+    [DataField]
     public string FaxName { get; set; } = "Unknown";
 
+    /// <summary>
+    /// Type of damage dealt when faxecuted.
+    /// </summary>
+    [DataField]
+    public ProtoId<DamageGroupPrototype> DamageType = "Brute";
+
+    /// <summary>
+    /// Amount of damage dealt when faxecuted.
+    /// </summary>
+    [DataField]
+    public int Damage = 300;
+
     /// <summary>
     /// Device address of fax in network to which data will be send
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite)]
-    [DataField("destinationAddress")]
+    [DataField]
     public string? DestinationFaxAddress { get; set; }
```

Comment:
This is going to break faxes on all of the maps due to the ID change so this needs reverting.